sdk/search/test.yml

```diff
@@ -0,0 +1,14 @@
+trigger: none
+
+jobs:
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+          ServiceDirectory: search
```

Comment:
Could we add a `maxParallel` configuration and set it to 2, maybe 3.

https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#matrix

FYI @tg-msft, this should help reduce the number of concurrent testing jobs happening at the same time.

Reply:
Set to 3 for initial check in